readme.md

```diff
@@ -16,10 +16,10 @@ You DON'T need to implement the card now, just the stars block.
 ![Stars](./reference/stars.png)
 ___
 - You can find star images in `images` folder
-- Reset browser's default `margin`
+- Reset browser's default `margin`  ?????
```

Comment:
????)

Reply:
Oops, it should not have been committed)